main/src/ca/uwaterloo/flix/tools/Packager.scala

```diff
@@ -547,4 +560,37 @@ object Packager {
       FileVisitResult.CONTINUE
     }
   }
+
+  /**
+    * The comparator which compares Path objects by a non platform-specific way.
+    * @see <a href="https://reproducible-builds.org/">Reproducible Builds</a>
+    */
+  class PathComparator extends Ordering[Path] {
+    /**
+      * Create an iterator that iterates name of path elements.
+      * e.g. `iterate(Paths.get("path/to/file"))` returns an iterator that iterates `["path", "to", "file"]`.
+      *
+      * @param p Path instance to iterate
+      * @return non-null iterator
+      */
+    private def iterate(p: Path): Iterator[String] =
+      p.iterator.asScala.map(
+        // Convert Path to String, to compare the name of path elements by a platform-independent way.
+        // According to Javadoc, the implementation of `Path.compareTo(Path)` is platform-specific.
+        Objects.toString
+      )
+
+    override def compare(l: Path, r: Path): Int = {
+      require(l.isAbsolute == r.isAbsolute)
```

Comment:
Would it be simpler just to change them both to absolute rather than potentially crashing?